src/main/Metadata.tsx

```diff
@@ -544,6 +545,8 @@ const Metadata: React.FC<{}> = () => {
                 onBlur: (e: any) => {blurWithSubmit(e, input)},
                 showError: showErrorOnBlur
               }}
+              leftArrowButtonText={i18next.t('metadata.calendar-prev-month')}
+              rightArrowButtonText={i18next.t('metadata.calendar-next-month')}
```

Comment:
This will display the month text for all left/right arrows, including the ones to switch between hours and minutes